### ISSUE TITLE
no-unused-variable: ignore-pattern does not work for imports

### ISSUE THREAD

**Issue by franekp** [palantir/tslint#3280]
### Bug Report

- __TSLint version__: 5.7.0
- __TypeScript version__: 2.5.2
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
import angular from 'angular'
```

with `tslint.json` configuration:

```json
{"rules": {"no-unused-variable": [true, {"ignore-pattern": "angular"}]}}
```

#### Actual behavior
ERROR: /.../example.ts[1, 1]: All imports are unused.
#### Expected behavior
No error.

**Comment by ajafff** [comment]
Fixed by #3187
Will be part of the next release.
